libs/resiliency/examples/async_replay.cpp

```diff
@@ -1,153 +1,85 @@
-//  Copyright (c) 2019 National Technology & Engineering Solutions of Sandia,
-//                     LLC (NTESS).
-//  Copyright (c) 2018-2019 Hartmut Kaiser
-//  Copyright (c) 2018-2019 Adrian Serio
-//  Copyright (c) 2019 Nikunj Gupta
+//  Copyright (c) 2019-2020 Nikunj Gupta
```

Comment:
What is your rationale of removing the copyrights? I think we should leave those in place.

Reply:
Ohh I removed the whole example for a simplified version of it. Looks like I mistakenly removed all the copyrights as well. Will correct it.